### PR TITLE
refactor(test): remove redundancy while mentioning keys

### PR DESCRIPTION
**Pull request by khareyash05**
## Problem

Not a breaking change, Lots of redundant code in integration-tests

## Changes

Removed redundant calling of public, secret keys and host URL by creating a single block of data to mention these keys

### Bump level

- [ ] Major
- [ ] Minor
- [x] Patch

### Libraries affected

- [x] All of them
- [ ] langfuse
- [ ] langfuse-node

### Changelog notes

- NA

### CODE REVIEW

File: integration-test/integration-utils.ts [hunk 1]
```diff
@@ -6,6 +6,12 @@ export const LF_HOST = process.env.LF_HOST ?? "http://localhost:3000";
 export const LF_PUBLIC_KEY = process.env.LF_PUBLIC_KEY ?? "pk-lf-1234567890";
 export const LF_SECRET_KEY = process.env.LF_SECRET_KEY ?? "sk-lf-1234567890";
 
+export const getKeys = {
```
Comment (ksjogo) [inline integration-test/integration-utils.ts:9]: While refactoring this, should we rename getKeys and getHeaders to make clear that they are not functions?
Or actually declare them as functions and invoke them?
At the moment the naming is a bit confusing.

Comment (khareyash05) [reply integration-test/integration-utils.ts:9]: I think renaming them would be a better options. Maybe something like `langfuseKeys`. WDYT @ksjogo ?

Comment (ksjogo) [reply integration-test/integration-utils.ts:9]: I am usually in favor of actually declaring things as functions as that's easier to refactor in the future if needs change.

Comment (khareyash05) [reply integration-test/integration-utils.ts:9]: True! Will do that!

Comment (khareyash05) [reply integration-test/integration-utils.ts]: Updated! Kindly review @ksjogo

**Comment by marcklingen** [comment]
thanks for the contribution and sorry for moving slowly on this. @maxdeichmann has added this in the meantime when doing a refactor of parts of the constructor logic

**Pull request closed without merging**